tests/std/tests/GH_003883_replace_asan_failure/env.lst

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation.

```

Comment:
Could we add the test case to the existing string ASan test so we don't need to duplicate this giant `env.lst`? If not, then can we avoid duplication by promoting this up a directory, naming it `usual_asan_something`, and having the three asan tests share it?

Reply:
The string and vector `env.lst`s differ somewhat. I think we should fuse this into the existing string test that already tests a bugfix. I can validate and push changes.

Reply:
Also, there was a missing `test.lst` change here.